dwave/optimization/generators.py

```diff
@@ -195,7 +197,7 @@ def bin_packing(weights: numpy.typing.ArrayLike,
     # Ensure that the weight for each bin does not exceed capacity
     for i in range(max_bins):
         bin_array = bins[i]
-        model.add_constraint((bin_array*weights).sum() <= capacity)
+        model.add_constraint((bin_array * weights).sum() <= capacity)
```

Comment:
I'd prefer to keep the diff minimal by not reformatting other parts of the code. It makes it harder to review and messes with the git history. Formatting could be done as a separate PR.

Reply:
Got it. That makes sense. I used PyCharm automated formatting and it has changed other lines as well. Will make sure that I undo changes to other lines in future.